organization/summit.md

```diff
@@ -0,0 +1,23 @@
+# Mentor Summit
+
+Every year Google promote a Summit
+and each [**mentoring**] organization,
+**in this case only NumFOCUS**,
+may send two (2) members to the Mentor Summit.
+
+NumFOCUS will nominate as Summit ambassador
+the lead administrator of the current year
+**and** the lead administrator of following year.
```

Comment:
I think it's a nice idea to allow the next year mentor to go as well. But I'm not sure how practical it is. For one only participants who sign up in the current year can come and Google wants people to sign up early in summer.  So there is half a year at least between a person agreeing to be the admin and actually having to admin. That's a lot of time for things to happen in life to not have time anymore.